api/tests/mocha/controllers/changes.js

```diff
@@ -122,7 +122,11 @@ describe('Changes controller', () => {
 
       this.then = promise.then.bind(promise);
       this['catch'] = promise['catch'].bind(promise);
-      this.then(result => complete(null, result), complete);
+      this.then(
+        result => complete(null, result), complete
+      ).catch(() => {
+        process.exit(1);
+      });
```

Comment:
I think it might be cleaner to call `complete` with the error, rather than just blowing up. I think this is what the old code was trying to do with the `, complete);` bit at the end of line 125 - it looks like a bad refactor at some time in the past.

Also, and this is a nit pick, with one line functions our style is to keep them on one line, eg:

```suggestion
      this
        .then(result => complete(null, result))
        .catch(err => complete(err));
```